src/lib/hooks/routing/useClientSideSmartOrderRouterTrade.ts

```diff
@@ -9,8 +9,13 @@ import useActiveWeb3React from '../useActiveWeb3React'
 import { getClientSideQuote } from './clientSideSmartOrderRouter'
 import { useRoutingAPIArguments } from './useRoutingAPIArguments'
 
-const protocols: Protocol[] = [Protocol.V2, Protocol.V3]
-const config = { protocols }
+const config = {
+  // Limit to only V2 and V3.
+  protocols: [Protocol.V2, Protocol.V3],
+
+  // Used to reduce latency while SOR is used client side. Default is 5
```

Comment:
Default is 5 for MAINNET, but 10 for OPTIMISM and 25 for ARBITRUM, so this may end up slowing down L2s. Can this be chain-specific? Should OPTIMISM stay at 10, or should we bump all L2s to 25?

Reply:
Instead of putting the default here, you should link to the code where it is defined (in case it changes in the future). Make sure to use a stable link, like https://github.com/Uniswap/smart-order-router/blob/309e6f6603984d3b5aef0733b0cfaf129c29f602/src/routers/alpha-router/config.ts#L83.

Reply:
Also, say what it does to reduce latency (this can be somewhat copypasta from the docs, as it's just more documentation), eg:

`Increase the minimum percentage of the input token to use for each route in a split route.`

(Original docs: https://github.com/Uniswap/smart-order-router/blob/309e6f6603984d3b5aef0733b0cfaf129c29f602/src/routers/alpha-router/alpha-router.ts#L260)